asset-transfer-basic/chaincode-java/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Basic asset transfer 
+
+This project demonstrates the use of the Java SDK, running a basic asset transfer contract using the "chaincode as a service" 
+pattern.
+
+
+## Setup 
+
+In this sample we will employ the [Kubernetes Test Network](../../test-network-k8s) to illustrate a scenario of 
+building, running, and debugging chaincode on a development workstation.  
+
+This project is also compatible with the legacy chaincode builder pipeline and the compose based test-network.
+For additional details, see the [End-to-end with the test-network](../../test-network/CHAINCODE_AS_A_SERVICE_TUTORIAL.md#end-to-end-with-the-the-test-network)
+documentation.
+
+## [Quickstart](../../test-network-k8s#quickstart)
+
+```shell
+export PATH=${PWD}/../../test-network-k8s:$PATH 
+
+network kind 
+
+network up 
+network channel create
+```
+
+```shell
+network chaincode deploy  ${PWD}
```

Comment:
A single deploy command is powerful.. and makes dev easier without a doubt. 

Thinking longer term about when this comes to deployments on multi-org network, can we make a single deploy work still?

Reply:
For "deploy" to work we will need to introduce some higher-order descriptors that articulate the structure of the overall network.  This would be a really nice addition, IMO.  At a meta level, the person configuring a Fabric network really should only focus on high-level constructs such as Organizations, Consortiums, Users (Identities), Channels, Chaincode, etc... and let some sort of software-automation package take responsibility for the minutia of configuration tasks.

Test-network[-k8s] is a good area for prototyping a high-level configuration target, but we should plan for alignment with some of the other "meta" Fabric configuration packages such as fabric-test-operator, minikube, *-k8s-operator, etc.   Ideally the '/network deploy' target can utilize whatever configuration specification is used by a common descriptor language.